ras_backstage/resources/reporting_units/get_reporting_unit.py

```diff
@@ -71,6 +72,15 @@ def add_collection_exercise_details(collection_exercises, reporting_unit, cases)
 def link_respondents_to_survey(respondents, survey):
     survey['respondents'] = []
     for respondent in respondents:
+
+        # Collect the collection case.
+        json = get_cases_by_business_party_id(respondent.get('id'))
+        try:
+            respondent['case_id'] = json[0].get('id')
```

Comment:
Can't we do this something like `respondent['case_id'] = json['id']`
And maybe name it something instead of `json`, maybe `cases` ?

Reply:
```
cases = case_controller.get_cases_by_business_party_id(respondent['id'])
        try:
            respondent['case_id'] = cases[0]['id']
```
This works for me, but I'm wondering about the cases[0] (0 is 'ACTIONABLE' and 1 'INACTIONABLE') I don't know what they're for.